README.md

```diff
@@ -1,17 +1,18 @@
-# shellcheck pre-commit hook
+# shellcheck mirror
 
-Hook for shellcheck with conda as a language.
+Mirror of shellcheck for pre-commit with conda as a language.
```

Comment:
What is this hook mirroring?

Reply:
https://github.com/koalaman/shellcheck-precommit when you broaden the term "mirroring"

Reply:
Also, this repo's name is pre-commit-_mirrors_-shellcheck

Reply:
Fixed in the next copier release